<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Academic</class>
 <widget class="QMainWindow" name="Academic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Academic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>651</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Here you will find several options regarding your academics here at BMCC. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Please select one of the options below:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="majorsButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show me the different majors and programs BMCC offers</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="advisementButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Advisement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePermitButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply for an ePermit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classesButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search for classes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="articulationButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Articulation Agreements</string>
    </property>
    <property name="icon">
     <iconset theme="face-monkey"/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
